README.md

```diff
@@ -30,7 +30,9 @@ The code of the benchmarking suite uses two main abstractions:
 
 ### Manual Usage
 
-The benchmarking suite can be built using `cargo build`.
+First set the `OPENTITAN_LIBS_PATH` environment variable to a folder containing the required opentitan libraries.
+The libraries that are required can be found in the `reproduce.py` script.
```

Comment:
Could you maybe enumerate upon the libraries that are required?

Reply:
Sure, 
I was just trying to avoid redundancy so that I do not miss updating them in the readme in potential future changes.